src/lib/broadcast.jl

```diff
@@ -45,18 +45,20 @@ function Base.reducedim_init(::typeof(identity), ::typeof(accum), A::AbstractArr
   Base.reducedim_initarray(A, region, nothing, Union{Nothing,eltype(A)})
 end
 
-trim(x, Δ) = reshape(Δ, ntuple(i -> size(Δ, i), Val(ndims(x))))
```

Comment:
I think doing this makes unbroadcast less generic, we don't need to define projections here afaict. Let's retain the current definition.

Reply:
What case exactly is not handled, if this is less generic?

Reply:
It restricts it to what can be handled by `_project` as opposed to simple sizes and lengths of arrays.